### PR TITLE
added plots of steer/speed/brake/throttle command+history (#53)

### PR DESCRIPTION
**Pull request by asimonov**

### CODE REVIEW

File: ros/src/waypoint_updater/show_waypoints.py [hunk 1]
```diff
@@ -187,7 +187,7 @@ def draw_plots(self):
         self.brake_axes.set_xlabel("Time", fontsize=8)
 
         self.steer_axes.clear()
-        self.steer_axes.set_ylim(-.1, .1)
+        #self.steer_axes.set_ylim(-.1, .1)
```
Comment (vguerra) [inline ros/src/waypoint_updater/show_waypoints.py]: just remove it :).

**Review by vguerra — APPROVED** [review]
lgtm.

**Review by yhoazk — APPROVED** [review]
Didn't tested it but code looks good.

**Comment by asimonov** [comment]
@avrabe Ralf, looks like some issues with docker:

https://travis-ci.org/Kairos-Automotive/carla-brain/builds/297190154#L624

it could not build dockerfile and then start container.
i did not touch anything around docker.

may be there are issues with dockerhub at the moment, not sure...

**Comment by asimonov** [comment]
the end result video:

https://youtu.be/4iODUu2ufNU

**Comment by avrabe** [comment]
@asimonov I had to remove the rviz package from docker as it caused the problems during the build. If you'd rebase the build should go through.

**Review by avrabe — APPROVED** [review]

**Review by dabavil — APPROVED** [review]

**Pull request merged**